tests/test_reuse.py

```diff
@@ -33,4 +31,7 @@ def test_reuse_in_widget(self):
         Widget templates want to reuse their own blocks.
         '''
         tmpl = get_template('inwidget')
-        output = tmpl.render(self.ctx)
+        try:
+            tmpl.render(self.ctx)
+        except:
```

Comment:
It's more so that the test fails instead of errors, no exveptions at all
are expected.

On Aug 14, 2016 2:31 AM, "Curtis Maloney" <notifications@github.com> wrote:

> In tests/test_reuse.py
> <https://github.com/funkybob/django-sniplates/pull/47#discussion_r74694024>
> :
>
> > @@ -33,4 +31,7 @@ def test_reuse_in_widget(self):
> >          Widget templates want to reuse their own blocks.
> >          '''
> >          tmpl = get_template('inwidget')
> > -        output = tmpl.render(self.ctx)
> > +        try:
> > +            tmpl.render(self.ctx)
> > +        except:
>
> Can we narrow down to specific exceptions here? I don't like Pokemon
> excepts.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/funkybob/django-sniplates/pull/47/files/3bc72abaec33abfddafe01c5a4f48352a096ff40#r74694024>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFQ01k04v5bti9aopF36XVKSSL-P2HOwks5qfmHQgaJpZM4JjqPT>
> .
>